plugin/Installer/DatabaseChanges.php

```diff
@@ -482,6 +487,42 @@ private function updateSortByUserDefinedDefault()
 				`sortByUserDefinedDefault` NOT LIKE '%#DESC'");
 	}
 
+	/**
+	 *
+	 */
+
+	public function migrationsDataSimilarEstates()
+	{
+		$defaultViewOptions = get_option('onoffice-default-view');
+		if(!empty($defaultViewOptions)){
+			add_option('onoffice-similar-estates-settings-view', '');
+			$old_data = [];
+			$old_data['enablesimilarestates'] = $defaultViewOptions->getDataDetailViewActive();
+
+			$dataDetailViewSimilarEstates = $defaultViewOptions->getDataViewSimilarEstates();
+			$old_data['radius'] = $dataDetailViewSimilarEstates->getRadius();
+			$old_data['same_kind'] = $dataDetailViewSimilarEstates->getSameEstateKind();
+			$old_data['same_maketing_method'] = $dataDetailViewSimilarEstates->getSameMarketingMethod();
+			$old_data['same_postal_code'] = $dataDetailViewSimilarEstates->getSamePostalCode();
+			$old_data['amount'] = $dataDetailViewSimilarEstates->getRecordsPerPage();
+			$old_data['similar_estates_template'] = $dataDetailViewSimilarEstates->getTemplate();
```

Comment:
The `fields` should be copied from the old view into the new one too.

Reply:
Also, variables for each individual value would be nicer than the current array `$old_data`.